Makefile

```diff
@@ -13,6 +13,9 @@ TESTER_PROGS_DIR = "contrib/tester-progs"
 # Extra flags to pass to test binary
 EXTRA_TESTFLAGS ?=
 
+BUILD_PKG_DIR ?= $(shell pwd)/build/
+BUILD_PKG_DIR_ARCH=$(BUILD_PKG_DIR)$(ARCH)
+
```

Comment:
do we need 2 variables for that? looks like BUILD_PKG_DIR could have the ARCH directly?

Reply:
Ok will try to remove!